src/View/Helper/ClickableLinksHelper.php

```diff
@@ -183,16 +183,14 @@ public function buildSentenceLink($sentenceId, $sentenceText = null)
                 that appears in logs to a more natural character  */
                 '#{sentenceId}'),
             array('sentenceId' => $sentenceId));
-        $model = TableRegistry::getTableLocator()->get('Sentences');
         return $this->Html->link(
             $linkText,
             array(
                 'controller' => 'sentences',
                 'action' => 'show',
                 $sentenceId
             ),
-            // $sentenceText could come as the empty string
-            array('title' => $sentenceText != null ? $sentenceText : $model->getSentenceTextForId($sentenceId))
+            ['title' => $sentenceText]
```

Comment:
The sentence text needs to be sanitized using `$this->_View->safeForAngular()`.

BTW: Shouldn't we start using `md-tooltip` for all new tooltips? (see #1221)

Reply:
Good catch about the XSS! :sunglasses: 
I also updated the code to use `md-tooltip`.